sw/device/tock/meson.build

```diff
@@ -18,6 +19,12 @@ rust_flags = '-C link-arg=-Tlayout.ld ' + \
 
 build_tock_cmd = meson.source_root() / 'util/build_tock.sh'
 
+if get_option('tock_local')
```

Comment:
I would suggest lifting this `get_option` to the root Meson file, like the rest of them.

Reply:
Done.